server/app/lib/actions/fusor/host/trigger_provisioning.rb

```diff
@@ -59,18 +60,18 @@ def assign_host_to_hostgroup(assignee_host, hostgroup)
                 find { |v| v.fact_name.name =~ /^macaddress_.*$/ }.
                 fact_name.name.split('_').last
 
-            ::Fusor.log.debug "the interface is #{interface}"
+            ::Fusor.log.info "the interface is #{interface}"
 
             network = hosts_facts.where(fact_names: { name: "network_#{interface}" }).first
 
-            ::Fusor.log.debug "the network is #{network.value}"
+            ::Fusor.log.info "the network is #{network.value}"
 
             if hostgroup.subnet
               hostgroup.subnet.network == network.value or
                   raise "networks do not match: #{hostgroup.subnet.network} #{network.value}"
             end
 
-            ::Fusor.log.debug "================ Finished validating discovered host facts ===================="
+            ::Fusor.log.info "================ Finished validating discovered host facts ===================="
           end
```

Comment:
-1 these are not usual pain points during a deployment. I would actually leave them as debug or maybe remove them.

+1 to the newly added line: 

```
:Fusor.log.info "Starting host #{input[:host_id]} deployment..."
```